### DISCUSSION TITLE
`batch` function link is dead in doc

### DISCUSSION THREAD

**Discussion by Baptistemontan** [leptos-rs/leptos#1624] [General]
https://github.com/leptos-rs/leptos/blob/bcfa430a40306d69077ebf41342f0c144d4d215a/docs/COMMON_BUGS.md?plain=1#L60

Link is dead, I'm not opening an issue as I don't know if it's related to v0.5.x update

**Comment by gbj** [comment]
Right, the state of the main branch right now reflects 0.5.0, so see https://docs.rs/leptos/0.5.0-beta2/leptos/fn.batch.html
